orc8r/cloud/go/services/configurator/servicers/southbound.go

```diff
@@ -41,6 +41,16 @@ func NewSouthboundConfiguratorServicer(factory storage.ConfiguratorStorageFactor
 	return &sbConfiguratorServicer{factory}, nil
 }
 
+type cloudSbConfiguratorServicer struct {
+	factory storage.ConfiguratorStorageFactory
+}
+
+func NewCloudSouthboundConfiguratorServicer(factory storage.ConfiguratorStorageFactory) (cfg_protos.CloudSouthboundConfiguratorServer, error) {
```

Comment:
Eh we can leave this one as-is, no need to make it protected -- it's the same data as the GetMconfig, it just happens to only be used internally

Reply:
Do you mean not to split the proto at all?!